src/OrchardCore/OrchardCore.DisplayManagement/Shapes/DateTimeShapes.cs

```diff
@@ -5,24 +5,28 @@
 using Microsoft.AspNetCore.Mvc.Rendering;
 using Microsoft.Extensions.Localization;
 using OrchardCore.DisplayManagement.Descriptors;
+using OrchardCore.Modules.Services;
 using OrchardCore.Modules;
 
 namespace OrchardCore.DisplayManagement.Shapes
 {
     public class DateTimeShapes : IShapeAttributeProvider
     {
-        private const string LongDateTimeFormat = "dddd, MMMM d, yyyy h:mm:ss tt";
+        //private const string LongDateTimeFormat = "dddd, MMMM d, yyyy h:mm:ss tt";
```

Comment:
@sebastienros Maybe you could tell me why we we're using a string like this here. When localizing this not works so well since the order of day, month, year can change per culture. Maybe the intent was to get more details on the date itself by adding "tt" in the end. Else, when removing this, dates are getting properly localized.